ui/analyse/src/study/chapterNewForm.ts

```diff
@@ -118,6 +118,7 @@ export function view(ctrl: StudyChapterNewFormCtrl): VNode {
   const currentChapter = ctrl.root.study!.data.chapter;
   const mode = currentChapter.practice ? 'practice' : (defined(currentChapter.conceal) ? 'conceal' : (currentChapter.gamebook ? 'gamebook' : 'normal'));
   const noarg = trans.noarg;
+  let isDefaultName = true;
```

Comment:
We should make no assumptions about when or how often `view()` is called. Therefore any state must be part of `ctrl` (or better `ctrl.vm`).

Reply:
True, I'm changing that rn